common/test/acceptance/tests/discussion/helpers.py

```diff
@@ -57,13 +57,11 @@ def disable_cohorting(self, course_fixture):
         """
         Disables cohorting for the current course fixture.
         """
-        course_fixture._update_xblock(course_fixture._course_location, {
-            "metadata": {
-                u"cohort_config": {
-                    "cohorted": False
-                },
-            },
-        })
+        url = LMS_BASE_URL + "/courses/" + course_fixture._course_key + '/cohorts/settings'  # pylint: disable=protected-access
+        data = json.dumps({'is_cohorted': False})
+        response = course_fixture.session.post(url, data=data, headers=course_fixture.headers)
```

Comment:
seems odd to have to perform this update over HTTP, is a reason we can't make a direct call to the relevant python API ?